e2e/cypress/support/assertions.js

```diff
@@ -12,3 +12,15 @@ export function beUnread(items) {
     expect(items).to.have.length(1);
     expect(items[0].className).to.match(/unread-title/);
 }
+
+// Asserts that an item in the channel sidebar is muted.
+export function beMuted(items) {
+    expect(items).to.have.length(1);
+    expect(items[0].className).to.match(/muted/);
+}
```

Comment:
That sounds like a good idea, but we've got so many tests that rely on the current behaviour that it might be a little too late to do that at this point, at least as part of this PR.
```
harrison-mm2:e2e harrison$ ag '#sidebarItem_' | wc -l
     183
```

Reply:
it may be good for a hacktoberfest campaign?